src/components/CourseById/Lessons/LockedLessons.jsx

```diff
@@ -0,0 +1,12 @@
+import { LessonsListItem, LockIcon } from './Lessons.module';
+
+const LockedLesson = ({ lesson }) => (
+  <LessonsListItem>
+    <p>
+      <LockIcon />
```

Comment:
Icon inside p? If the icon is svg, this would be a bad semantic. Use div here instead.